docs/docs/default-actions.mdx

```diff
@@ -93,3 +108,46 @@ class ActionSessionStart(Action):
 
         return events
 ```
+
+## action_default_fallback
+
+This action undoes the last user-bot interaction and sends the `utter_default` response if it is defined.
+It is triggered by low action prediction confidence, if you have this [fallback mechanism](./fallback-handoff.mdx) enabled.
+
+## action_deactivate_loop
+
+This action deactivates the active loop and resets the requested slot. This is used when
+[handling unhappy paths in forms](./forms.mdx#writing-stories--rules-for-unhappy-form-paths).
+
+## action_revert_fallback_events
+
+:::caution Deprecated!
+The `action_revert_fallback_events` action is deprecated since the 2.0 release.
+The [`action_two_stage_fallback`](./default-actions#action_two_stage_fallback) action
+now handles all [two-stage fallback behaviour](./fallback-handoff.mdx)
+:::
+
+## action_two_stage_fallback
+
+This is a fallback loop that can be used to handle low NLU confidence. Read more about
+[handling low NLU confidence](./fallback-handoff#handling-low-nlu-confidence).
+
+## action_default_ask_affirmation
+
+This action is used by the `action_two_stage_fallback` loop. It asks the user to confirm
+the intent of their message. This action can be customised to be more personalised
+to your specific use case.
+
+<!-- TODO: should we show how to personalise this here? -->
```

Comment:
@erohmensing thoughts?

Reply:
Oops I missed this one. I think maybe we could link to the actions file (not a specific line bc itll move) but "for example the rasa demo bot <link> overrides `action_default_ask_affirmation` to provide nice button titles for intents, taking entities into account

or something like that?.